app/src/main/java/ch/epfl/favo/view/tabs/favorList/searchBarCoordinator.java

```diff
@@ -0,0 +1,136 @@
+package ch.epfl.favo.view.tabs.favorList;
+
+import android.content.Context;
+import android.graphics.Point;
+import android.util.Log;
+import android.view.Display;
+import android.view.View;
+import android.widget.ListView;
+import android.widget.SearchView;
```

Comment:
Now that we have a toolbar you may want to integrate the search view there for simplicity but it's not really necessary at this point. If you want, you can take inspiration from the favors page fragment of the current master.

Reply:
I plan to do it when I merge with master